sass/components/_chart-types.scss

```diff
@@ -30,10 +31,88 @@
     }
 }
 
+.sc-chart-type {
+    display: flex;
+    padding: 24px;
+    justify-content: space-between;
+    @include themify($themes) {
+        background: themed('ChartTypeBg');
+    }
+
+    &__item {
+        $self: &;
```

Comment:
Looks like $self is not used anywhere. Can we remove this?

Reply:
yeah, seems I removed the related code in the refactoring but forget to fix this. thanks.